module2/src/com/epam/onedimensionalarrays/SumCalculatorOfNumbersDivisibleByK.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.onedimensionalarrays;
+
+public class SumCalculatorOfNumbersDivisibleByK {
+    public static int countSum(int[] array, int divider) {
+        int sum = 0;
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] % divider == 0) {
+                sum += array[i];
+            }
+        }
```

Comment:
You can use here for-each:
```suggestion
        for (int number : array) {
            if (number % divider == 0) {
                sum += number;
            }
        }
```